### ISSUE TITLE
Off-Canvas Triggers Not Firing in Foundation 6

### ISSUE THREAD

**Issue by scott-schafer** [zurb/foundation-sites#9761]
I am trying to create an Off-Canvas Menu for a project and I can not get any of the examples on your website to work locally, using the examples I found in the foundation docs for version 6. I have tried to recreate the examples and I can not get the panel trigger to fire on anything that is using Foundation 6. I even block copied the HTML over and they still do not work. I was able to create my own off canvas menu using Foundation 5 and that worked fine.

An example is below and the link to the example page that I am referring to:

http://foundation.zurb.com/sites/docs/off-canvas.html

<!doctype html>
<html class="no-js" lang="en">
	<head>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
		<title>Multi_side_test</title>
		<link rel="stylesheet" href="css/foundation.css"/>
		<link rel="stylesheet" href="css/app.css"/>
		<script src="js/vendor/modernizr.js"></script>

	</head>
	<body>

<button type="button" class="button" data-toggle="offCanvasLeft1">Open Left</button>
<button type="button" class="button" data-toggle="offCanvasRight1">Open Right</button>
<button type="button" class="button" data-toggle="offCanvasTop1">Open Top</button>
<button type="button" class="button" data-toggle="offCanvasBottom1">Open Bottom</button>

<div class="row column">
  <div class="off-canvas-wrapper">
    <div class="off-canvas position-left" id="offCanvasLeft1" data-off-canvas>
      
    </div>
    <div class="off-canvas position-right" id="offCanvasRight1" data-off-canvas>
      
    </div>
    <div class="off-canvas position-top" id="offCanvasTop1" data-off-canvas>
      
    </div>
    <div class="off-canvas position-bottom" id="offCanvasBottom1" data-off-canvas>
      
    </div>
    <div class="off-canvas-content" data-off-canvas-content>
      <img src="http://placehold.it/300x300" class="" height="" width="" alt="">
    </div>
  </div>
</div>
	
		<script src="js/vendor/jquery.js"></script>
		<script src="js/foundation.min.js"></script>
		<script>
			$(document).foundation();
		</script>

</body>
</html>

**Comment by IamManchanda** [comment]
1. If this is a support(plz halp) request and not a bug fix request then plz ask here => http://foundation.zurb.com/forum 
2. If its a bug, then please provide Test case (codepen) for any kind of bug fixes! 
You can use/fork one of these two for the same 
   - http://codepen.io/IamManchanda/pen/vgemXR - Foundation 6 without Flexbox
   - http://codepen.io/IamManchanda/pen/xgXdRq - Foundation 6 with Flexbox

**Comment by brettsmason** [comment]
@scott-schafer We would need to see your markup in order to help. However can you check that the ID of the off canvas and the `data-toggle` of the button is the same? eg:

```
<div class="off-canvas position-left" id="offCanvasLeft" data-off-canvas>
</div>
```
```
<button type="button" class="button" data-toggle="offCanvasLeft">Open</button>
```
Failing that I'd need to see your code.

**Comment by scott-schafer** [comment]
Hi Brettsmason,
Below is the code that you asked for. Thank You for helping out.
```
<!DOCTYPE html>
<!--[if IE 9]>
<html class="lt-ie10" lang="en" >
<![endif]-->
<html class="no-js" lang="en" >
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">

	<title>Test_8</title>
	<link rel="stylesheet" href="css/normalize.css">
	<link rel="stylesheet" href="css/foundation.css">
	<script src="js/vendor/modernizr.js"></script>
</head>

<body>

	<button type="button" class="button" data-toggle="offCanvasLeft">Open Menu</button>

	<div class="off-canvas position-left" id="offCanvasLeft" data-off-canvas>


				<!-- Menu -->
	<ul class="vertical menu">
	  <li><a href="#">Foundation</a></li>
	  <li><a href="#">Dot</a></li>
	  <li><a href="#">ZURB</a></li>
	  <li><a href="#">Com</a></li>
	  <li><a href="#">Slash</a></li>
	  <li><a href="#">Sites</a></li>
	</ul>

	</div>
	   
	<div class="off-canvas-content" data-off-canvas-content>
	    <div class="row">
		<div class="medium-6 columns">Hey there Dude!</div>
		<div class="medium-6 columns">Some sample text here.</div> 
		</div>     

	<h3>Artisanal filler text for your project.</h3>


	<p>Bushwick tote bag pop-up sartorial. Next level vinyl pug venmo, hashtag craft beer stumptown health goth. Aesthetic austin craft beer, meditation cliche taxidermy portland next level vegan slow-carb. Migas skateboard pork belly green juice, keffiyeh church-key health goth poke unicorn schlitz disrupt cornhole. Semiotics etsy swag, irony ethical offal hella cardigan synth keffiyeh mlkshk meh small batch portland. Cold-pressed vexillologist you probably haven't heard of them, keffiyeh narwhal marfa neutra art party williamsburg try-hard fixie irony butcher tbh umami. Hexagon chia taxidermy umami.</p>

	</div>		
	</div>



					<!-- Exit Overlay -->


	<a class="exit-off-canvas"></a>
	</div>


	<script src="js/vendor/jquery.js"></script>
	<script src="js/foundation.js"></script>
	<script>
		$(document).foundation();
	</script>

</body>
</html>
```

**Comment by scott-schafer** [comment]
Hi Brett,

Issue resolved, what a dope I am! It turns out that dummy me had the script source code pointing to the wrong place!

It was:
```
<script scr=”js/foundation.js </script>
```
Instead of
```
<script scr=”js/vendor/foundation.js </script>
```
Let this be a lesson to all that when in the throws of solving coding issues it is a good idea to take a break to clear and reset your mind. It was such an obvious thing that once checked early on it became a given. A given that at some point got moved and the code was never updated. But in my mind I never gave it a second thought. I only discovered it after I installed the component Javascript version of 6.3.0 from Github and that worked, so I wanted to know why the individual component Javascript worked and the complete foundation Javascript did not. That is when I discovered the problem.

**I am Sooooo dumb! ;-)**

Anyhow, I am just glad that I can use Foundation 6 on this project. It is my first experience with Foundation and I always like to learn on the latest and the greatest.



Thank You for Your Time Brett !!
